src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,5 @@
+$clock-height: 80vmin;
+$clock-width:80vmin;
+$hand-width: 2vmin;
+$border-color: 1px solid rgb(0, 255, 17);
+$border-radius: 40%;
```

Comment:
Also some of these like this one here are too much, your css code would be more readable if only colors were kept as variables.